src/_includes/examples/accessible.html

```diff
@@ -0,0 +1,96 @@
+<section id="accessible">
+  <h2 class="page-header">
+    Accessibility-Minded
+  </h2>
+  <div class="row">
+    <div class="col-md-3 col-sm-4">
+      <p>
+        <a class="btn btn-default" href="path/to/settings">
+          <i class="fa fa-cog" aria-hidden="true"><span class="sr-only">Settings</span></i>
```

Comment:
adding `sr-only` to something whose parent is has `aria-hidden="true"` will render it invisible to screen readers. You can fix this in two ways. Either (preferred)
```
<a class="btn btn-default" href="path/to/settings" title="Settings">
          <i class="fa fa-cog" role="presentation"></i>
</a>
```
or (lots of redundancy)
```
<a class="btn btn-default" href="path/to/settings" title="Settings">
          <i class="fa fa-cog" aria-hidden="true"><span class="sr-only">Settings</span></i>
</a>
```
This will also ensure that the anchor will work for systems like Dragon